ConfigSpace/hyperparameters.pyx

```diff
@@ -331,7 +333,38 @@ cdef class UniformFloatHyperparameter(FloatHyperparameter):
     def __init__(self, name: str, lower: Union[int, float], upper: Union[int, float],
                  default_value: Union[int, float, None] = None, q: Union[int, float, None] = None, log: bool = False,
                  meta: Optional[Dict]=None) -> None:
+        """
+        Creates a float hyperparameter with values sampled from a uniform distribution with values
```

Comment:
Please do not write "Crease a ...", but just "a float hyperparameter" and add the explanation below.

Same holds for all other hyperparameters.